hackclub.com.yaml

```
@@ -477,6 +477,10 @@ cca:
 cccs:
   type: CNAME
   value: cname.vercel-dns.com.
+cchshackclub:
+- ttl: 1
+  type: CNAME
+  value: cchshackclub.github.io
```

```suggestion
  value: cchshackclub.github.io.
```
just to make OctoDNS happy!